libs/sdk-ui-filters/src/AttributeFilter/AttributeFilterButton.tsx

```diff
@@ -83,6 +96,11 @@ export interface IAttributeFilterButtonOwnProps {
      * loading attribute elements.
      */
     onError?: (error: any) => void;
+
+    /**
+     * Optionally customize attribute filter with a component to be rendered if attribute elements loading fails
+     */
+    FilterError?: React.ComponentType<{ error?: any }>;
```

Comment:
Typically we use this for the error components (both name and type it lives in sdk-ui):

```ts
    ErrorComponent?: React.ComponentType<IErrorProps>;
```

Reply:
Agreed not to follow as AttributeFilter has the prop with the same name and type